packages/data-transport-layer/src/services/server/service.ts

```diff
@@ -125,10 +125,17 @@ export class L1TransportServer extends BaseService<L1TransportServerOptions> {
    * Initialize Sentry and related middleware
    */
   private _initSentry() {
-    Sentry.init({
+    const sentryOptions = {
       dsn: this.options.sentryDsn,
       release: this.options.release,
       environment: this.options.ethNetworkName,
+    }
+    this.logger = new Logger({
+      name: this.name,
+      sentryOptions,
+    })
```

Comment:
this will be a different logger from what the base services uses, since this isn't passed in while initializing the options, correct?

not necessarily a problem but wanted to highlight

Reply:
I think it actually overwrites the existing one.

Reply:
I did it this way because otherwise it would require refactoring the `_initSentry` method which I didn't want to include in the scope of this PR.